ddtrace/profiling/event.py

```diff
@@ -16,7 +16,7 @@ class Event(object):
 
     __slots__ = ("timestamp",)
 
-    def __init__(self, timestamp=compat.time_ns()):
+    def __init__(self, timestamp=time.time_ns()):
```

Comment:
Is this correct? `time.time_ns()` will only be evaluated once at module load time, not every time the constructor is called.

Reply:
Good shout. This shouldn't be a big issue because generally the profiler is a singleton and it is booted at start-up. The same _kinda_ applies to tests (modulo a few delays perhaps). However, definitely something to keep in mind, just in case.

Reply:
oh, good catch. That sounds like a bug to me but I can't find any location that uses `timestamp`. I'll try removing it.

Reply:
https://github.com/DataDog/dd-trace-py/pull/10864